source/win-main/App.vue

```diff
@@ -699,4 +699,23 @@ export default {
 
 <style lang="less">
 //
+ /* width */
+::-webkit-scrollbar {
+  width: 10px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  background: rgba(255, 255, 255, 0.05);
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: rgba(255, 255, 255, 0.3);
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
```

Comment:
If I see this file correctly, you are not at all namespacing these custom styles, right? This would (on macOS) lead to scrollbars that are not automatically hiding, and on Windows it might make them look weird.

Generally, we should refrain from customizing the scrollbars since most operating systems have their built-in styles. But if this is better, what we must do at least is make sure these **do not apply to macOS**, since there scrollbars generally automatically hide and the native scrollbars work in both dark and light mode. So for macOS this won't work.